src/components/InputForm.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from 'styled-components'
+import InputForm from './InputForm'
+import StoryRouter from 'storybook-react-router'
+import { storiesOf } from '@storybook/react'
+import { linkTo } from '@storybook/addon-links'
+import testMock from './teasMock.json'
+import { action } from '@storybook/addon-actions'
+
+export default {
+  title: 'InputForm',
+  component: InputForm,
+}
+
+storiesOf('InputForm', module)
+  .addDecorator(
+    StoryRouter({
+      '/create': linkTo('LinkedStories', 'InputForm'),
+    })
+  )
+  .add('InputNewTea', () => (
+    <AppGrid>
+      <InputForm tea={testMock[0]}>
+        <StyledAddButton type="submit" onClick={action('clicked')}>
```

Comment:
Brauchst du den Button hier extra mit hinschreiben? Bei mir hatte das action auf den Button auch immer funktioniert, wenn ich die ganze Komponente so wie sie war importiert hatte. Müsstest du nicht so sogar einen zweiten Button in der Story sehen? Kannst ja mal ausprobieren was passiert, wenn du den Button hier weglässt und nur die Inputform importierst.